css/style.css

```diff
@@ -248,75 +296,48 @@ padding: 10px 20px;
     padding: 20px 0;
   }
 
-  .first-project {
-    background-image: url(images/clear-bg.png);
+  .project {
     background-repeat: no-repeat;
     background-position: center;
     background-size: cover;
     height: 400px;
     display: grid;
     grid-template-rows: repeat(1, 1fr);
     align-items: flex-end;
+  }
+
+  .first-project {
+    background-image: url(images/clear-bg.png);
```

Comment:
- [ ] You need to make sure the background images are displaying correctly. Please fix the URLs in `background-image` 
In order to fix you need to add `""` around the image URLs and add `/` at the start of URLs
### Do the following to all the background image URLs in your project:
**For example:**
In your `style.css` file at `line 108`  `background-image: url(images/illustration-header-mobile.png);` should be `background-image: url("/images/illustration-header-mobile.png");`